packages/web/src/components/notification/NotificationPanel.tsx

```diff
@@ -58,7 +58,7 @@ export const NotificationPanel = ({
     isAllPending: isPending,
     isError,
     isFetchingNextPage
-  } = useNotifications()
+  } = useNotifications({ enabled: isOpen })
```

Comment:
do we want this? i feel like prefetching is good

Reply:
good q, it's so fast now that it's prob fine to prefetch, will revert